python/README.md

```diff
@@ -1193,11 +1193,14 @@ policy = await cdp.policies.delete_policy(id="__POLICY_ID__")
 
 We currently support the following policy rules:
 
-- `SendEvmTransactionRule`
-- `SignEvmHashRule`
-- `SignEvmMessageRule`
-- `SignEvmTransactionRule`
-- `SignSolanaTransactionRule`
+- [SendEvmTransactionRule](https://docs.cdp.coinbase.com/api-reference/v2/rest-api/policy-engine/create-policy#option-2)
+- [SignEvmHashRule](https://docs.cdp.coinbase.com/api-reference/v2/rest-api/policy-engine/create-policy#option-6)
+- [SignEvmMessageRule](https://docs.cdp.coinbase.com/api-reference/v2/rest-api/policy-engine/create-policy#option-3)
+- [SignEvmTransactionRule](https://docs.cdp.coinbase.com/api-reference/v2/rest-api/policy-engine/create-policy#option-1)
+- [SignSolanaTransactionRule](https://docs.cdp.coinbase.com/api-reference/v2/rest-api/policy-engine/create-policy#option-5)
+- [SendUserOperationRule](https://docs.cdp.coinbase.com/api-reference/v2/rest-api/policy-engine/create-policy#option-8)
```

Comment:
thanks for this. can we include `SendSolanaTransaction`?

Reply:
The SendSolanaTransaction link will be correct once the mintlify docs are updated with the latest spec